core/src/test/java/org/elasticsearch/search/aggregations/metrics/scripted/ScriptedMetricAggregatorTests.java

```diff
@@ -117,7 +118,7 @@ public void testScriptedMetricWithoutCombine() throws IOException {
                 assertEquals(AGG_NAME, scriptedMetric.getName());
                 assertNotNull(scriptedMetric.aggregation());
                 Map<String, Object> agg = (Map<String, Object>) scriptedMetric.aggregation();
-                assertEquals(numDocs, ((List<Integer>) agg.get("collector")).size());
+                assertEquals(numDocs, ((List<Double>) agg.get("collector")).size());
```

Comment:
I think we should test cases with score separately from those without scores and retain both types of test.